CMakeLists.txt

```diff
@@ -35,7 +35,7 @@ if (BUILD_SDF)
   ign_configure_project(
     NO_IGNITION_PREFIX
     REPLACE_IGNITION_INCLUDE_PATH sdf
-    VERSION_SUFFIX pre1)
+    VERSION_SUFFIX)
```

Comment:
We shouldn't change this

Reply:
fixed via amended commit in https://github.com/ignitionrobotics/sdformat/pull/966/commits/1943c11820c1f1fdb7c88457ce83f30c4f790327